doc/release-notes/6497-migrate-api.md

```diff
@@ -0,0 +1,7 @@
+# Release Highlights
+
+### Dataset Migration API (Experimental)
+
+Datasets can now imported/updated following the format of an OAI-ORE export (RDA-conformant Bags), allowing for not only easier migration from one Dataverse installation to another, but also for better support of import from other systems. This experimental endpoint also allows keeping the existing persistent identifier and publication dates.
```

Comment:
The DOI authority has to match, right?

Reply:
will add

Reply:
added this in https://github.com/IQSS/dataverse/pull/7504/commits/45e585334d37be8a7204e7984fa7173412db3689